parquet-column/src/main/java/org/apache/parquet/column/values/bloomfilter/BlockSplitBloomFilter.java

```diff
@@ -371,4 +371,32 @@ public boolean equals(Object object) {
     }
     return false;
   }
+
+  private boolean checkCompatibility(BlockSplitBloomFilter that) {
+    return this.maximumBytes == that.maximumBytes && this.hashStrategy == that.hashStrategy;
```

Comment:
I think we need to check the algorithm as well.

Reply:
You re right.